### PR TITLE
Added 256bitPadded.code for sha256 on field[256] -> field[256] 

### PR DESCRIPTION
**Pull request by alexeiZamyatin**
Added 256bitPadded.code to /stdlib/hashes/sha256.
Allows to call sha256 on field[256] input, generating field[256] as output (no unpacking / packing).

Useful when computing Bitcoin's double-sha256.

### CODE REVIEW

File: zokrates_stdlib/stdlib/hashes/sha256/256bitPadded.code [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import "./512bit.code" as sha256 
+
+// A function that takes 1 field[256] array as input 
+// and returns the sha256 full round output as an array of 256 field elements.
+def main(field[256] a) -> (field[256]):
+
+    // Hash is computed on the full 256bit block size
```
Comment (Schaeff) [inline zokrates_stdlib/stdlib/hashes/sha256/256bitPadded.code]:
```suggestion
    // Hash is computed on 256bits of input
```

File: zokrates_stdlib/stdlib/hashes/sha256/256bitPadded.code [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import "./512bit.code" as sha256 
+
+// A function that takes 1 field[256] array as input 
+// and returns the sha256 full round output as an array of 256 field elements.
+def main(field[256] a) -> (field[256]):
+
+    // Hash is computed on the full 256bit block size 
+    // padding does not fit in the primary block
```
Comment (Schaeff) [inline zokrates_stdlib/stdlib/hashes/sha256/256bitPadded.code]:
```suggestion
    // padding fits in the remaining 256 bits of the first block
```

**Comment by alexeiZamyatin** [comment]
done, sorry for the delay!

**Comment by Schaeff** [comment]
All good thanks! I'll add a test in another branch.

**Pull request merged**